lib-sql/tables.sql

```diff
@@ -244,6 +244,23 @@ CREATE UNIQUE INDEX idx_postcode_id ON location_postcode USING BTREE (place_id)
 CREATE INDEX idx_postcode_geometry ON location_postcode USING GIST (geometry) {{db.tablespace.address_index}};
 GRANT SELECT ON location_postcode TO "{{config.DATABASE_WEBUSER}}" ;
 
+-- Table to store location of entrance nodes
+DROP TABLE IF EXISTS place_entrance;
+CREATE TABLE place_entrance (
+  place_id BIGINT NOT NULL,
+  osm_node_id BIGINT NOT NULL,
+  type TEXT NOT NULL,
+  geometry GEOMETRY(Point, 4326) NOT NULL
+  );
```

Comment:
This may be  a stupid idea but if you structure the table like this instead:
```
   place_id BIGINT NOT NULL,
   entrance_info JSONB
```
and then store one line per place with the entrance information already processed in the way you need it in the API calls below, you might be able to avoid all the complications around json-compatibility for sqlite.

Reply:
Yeah that's not a bad idea. We're not querying against the location data here anyway, so it might as well just live in a JSONB column. And yeah, this sqlite compatibility has grown into a mess.